tools/aws_lab_setup/roles/aws_workshop_login_page/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+workshop_dns_zone: "rhdemo.io"
```

Comment:
I'm not OK with this. Who manages this domain? What if someone in the community wants to use this without running things under a domain?

Reply:
This only matters if create_login_page: is set to true, this is for instructors only, the default is false and this won't be used. This is a hard requirement from the tiger team to effectively teach classes. We have tested different route53 domains so for example Gerald can still use redhatgov.io. I can however document this more clearly in the docs, would prefer this to be a separate issue though because I am on a time crunch.